src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
So pretend we were using something other than Direct Line. Is this something that would still need to be communicated between the bot and WebChat? If so, it legitimately belongs in IBotConnection in some form. But the other Conversation stuff - conversationId, token - is specific to Direct Line and does not.

You could just add it as a field, but you'd have to be careful when you reference it. It shouldn't happen until after connectionStatus$ first changes to ConnectionStatus.Online.

If it's only used for reading aloud an activity that's already been sent from the bot, then this may not be an issue, but I'm thinking of the "load prior history" feature which we've talked about elsewhere. In that case we have a valid activity but we may not yet be connected.